addons/html_builder/static/src/website_builder_action.js

```diff
@@ -34,10 +34,11 @@ export class WebsiteBuilder extends Component {
     setup() {
         this.orm = useService("orm");
         this.websiteContent = useRef("iframe");
+        this.state = useState({ isEditing: false, mobileContext: { isMobile: false } });
         useSubEnv({
+            mobileContext: this.state.mobileContext,
```

Comment:
Maybe overkill to pass it in the env ? Maybe wait to have a real use case ? Props is enough in this case no ?